dace/frontend/python/parser.py

```diff
@@ -653,6 +656,8 @@ def load_sdfg(self, path: str, *args, **kwargs) -> None:
         # Update SDFG cache with the SDFG (without a compiled version)
         self._cache.add(cachekey, sdfg, None)
 
+        return sdfg, cachekey
+
     def load_precompiled_sdfg(self, path: str, *args, **kwargs) -> None:
```

Comment:
This should also have the output type `None` removed from the signature (or updated to reflect what is actually returned.